packages/nux/src/components/inline-tip/README.md

```diff
@@ -0,0 +1,34 @@
+InlineTip
+=========
+
+`InlineTip` is a React component that renders a single _tip_ on the screen. It has the same appearance as a `Notice`. Each tip is uniquely identified by a string passed to `tipId`.
```

Comment:
Could link to `Notice` here and probably mention it's from the components package.

Reply:
👌 8a25104e4